README.md

```diff
@@ -1,6 +1,6 @@
 # Consent Ontology Services
 
-[![Build Status](https://travis-ci.com/broadinstitute/consent-ontology.svg?token=3ve6QNemvC5zpJzsoKzf&branch=develop)](https://travis-ci.com/broadinstitute/consent-ontology)
+[![Build Status](https://travis-ci.org/DataBiosphere/consent-ontology.svg?token=3ve6QNemvC5zpJzsoKzf&branch=develop)](https://travis-ci.org/DataBiosphere/consent-ontology)
```

Comment:
remove the token, I think. If I look at https://github.com/DataBiosphere/consent-ontology/tree/GAWB-3024 I can see the badge is broken.